tox.ini

```diff
@@ -60,10 +61,10 @@ setenv =
 commands =
     pytest -rfE --doctest-modules autonomy tests/ --cov=autonomy --cov-report=html --cov=packages/valory --cov-report=xml --cov-report=term --cov-report=term-missing --cov-config=.coveragerc {posargs}
 
-
-[testenv:py3.10]
+[testenv:py3.10-linux]
 basepython = python3.10
 usedevelop = True
+platform=^linux$
```

Comment:
this gets evaluated against sys.platform and so `tox` works out of the box with the platform specific commands